src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: $card-main-color;
+  transition: transform 0.3s;
+
+  &__title {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    margin-bottom: 4px;
+    transition: color 0.3s;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    display: flex;
+    margin-bottom: 40px;
+
+    img {
+      height: 134px;
+      width: 160px;
+      margin: 0 auto;
+    }
+  }
+
+  &__code {
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
```

Comment:
Ensure that the variable `$secondary-text-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.